domains/client/block-preprocessor/src/lib.rs

```diff
@@ -228,7 +229,7 @@ where
                         .map(|(signer, tx)| {
                             (
                                 signer.clone(),
-                                HeaderHashingFor::<Block::Header>::hash_of(tx),
+                                ExtrinsicDigest::new::<LayoutV1<BlakeTwo256>>(tx.encode()),
```

Comment:
Again, do not use bare hadh function. Always use the hash function from the DomainHeader

Reply:
It is used to keep consistency with the `ExtrinsicDigest` in `generate_invalid_domain_extrinsics_root_proof`, but yes, I will update both places to use the `DomainHeader`